src/main/webapp/js/common.es6

```diff
@@ -860,36 +953,22 @@ function sanitizeForJs(rawString) {
  *                             Example- '.panel-body, #panel-data, .sub-container'
  */
 function highlightSearchResult(searchKeyId, sectionToHighlight) {
-    var searchKey = $(searchKeyId).val().trim();
+    const searchKey = $(searchKeyId).val().trim();
     // split search key string on symbols and spaces and add to searchKeyList
-    var searchKeyList = [];
+    let searchKeyList = [];
     if (searchKey.charAt(0) === '"' && searchKey.charAt(searchKey.length - 1) === '"') {
         searchKeyList.push(searchKey.replace(/"/g, '').trim());
     } else {
-        $.each(searchKey.split(/[ "'.-]/), function() {
+        $.each(searchKey.split(/[ "'.-]/), function () {
             searchKeyList.push($.trim(this));
         });
     }
     // remove empty elements from searchKeyList
-    searchKeyList = searchKeyList.filter(function(n) {
-        return n !== '';
-    });
+    searchKeyList = searchKeyList.filter(n => n !== '');
     $(sectionToHighlight).highlight(searchKeyList);
 }
 
 /**
- * Polyfills the String.prototype.includes function finalized in ES6 for browsers that do not yet support
- * the function.
- */
-/* eslint-disable no-extend-native */
-if (!String.prototype.includes) {
```

Comment:
I don't think we've supported automatic polyfills yet. @li-kai correct me if I'm wrong.
Which means this polyfill still needs to be around.